src/rendering/renderers/shared/background/BackgroundSystem.ts

```diff
@@ -95,9 +93,14 @@ export class BackgroundSystem implements System
      */
     public init(options: BackgroundSystemOptions): void
     {
-        options = { ...defaultBackgroundOptions, ...options };
+        options = { ...BackgroundSystem.defaultOptions, ...options };
 
         this.clearBeforeRender = options.clearBeforeRender;
+        if (options.background !== undefined)
+        {
+            deprecation(v8_0_0, 'background is deprecated, please use backgroundColor');
```

Comment:
I prefer background. I added this because backgroundColor felt so verbose. I'd rather remove that.